src/pybind/mgr/dashboard/tox.ini

```diff
@@ -84,7 +84,7 @@ wrap_length = 80
 [pylint]
 # Allow similarity/code duplication detection
 jobs = 1
-dirs = . api controllers plugins services tests
+dirs = . api controllers plugins services tests ../../../../qa/tasks/mgr/dashboard
```

Comment:
Hey @dorindabassey,

Thank you for your contribution! These changes were part of this already merged pull request https://github.com/ceph/ceph/pull/37568. Does your pull request possibly need a rebase?

Reply:
my Apologies, i'd do a rebase thanks.